### PR TITLE
Update documents for DoIP

### PR DESCRIPTION
**Pull request by huavanthong**
Hi Thomas,

I would like to update the documents for DoIP at ticket https://github.com/test-fullautomation/robotframework-doip/issues/11
Please help me review it and let me know if you have any comments.

Thank you and best regards,
Thong Hua

### CODE REVIEW

File: packagedoc/additional_docs/pictures/diagnostic-process-in-system.png
Comment (test-fullautomation) [inline packagedoc/additional_docs/pictures/diagnostic-process-in-system.png:1]: Hi @huavanthong ,
what is the source of this three images?
Did you make them?
Thank you,
 Thomas

Comment (huavanthong) [reply packagedoc/additional_docs/pictures/diagnostic-process-in-system.png:1]: Hi Thomas, 

Currently, I referred to these images from the internet, but I also intend to create my own workflow and image to make it clearer.

Thank you,
Thong Hua

Comment (test-fullautomation) [reply packagedoc/additional_docs/pictures/diagnostic-process-in-system.png:1]: Hi Thong,
hm... we can only use the images if the licence allows it. What licence do they have?
Btw: The same applies for texts. Copying is not allowed except of licence allows it...
Can you check this please?
Thank you,
 Thomas

Comment (huavanthong) [reply packagedoc/additional_docs/pictures/diagnostic-process-in-system.png:1]: Hi Thomas,

Thank you for the reminder. I've checked it and found that there are some copyrights that I don't recognize such as:
* © 2024 Embitel. All Rights Reserved - from https://www.embitel.com
* © Keysight Technologies 2000–2024 - from https://www.keysight.com

Right now, I will create my own images instead it.

Thank you and best regards,
Thong Hua

Comment (test-fullautomation) [reply packagedoc/additional_docs/pictures/diagnostic-process-in-system.png:1]: Very good!
Please make for all your own pictures. Except of you find explicitly written that it's free for use.
Please also take care with copy&paste of texts!

Comment (huavanthong) [reply packagedoc/additional_docs/pictures/diagnostic-process-in-system.png:1]: Hi Thomas,

I have updated the images and summarized the text as concisely as possible. Could you please review it and let me know if there are any unclear sections in the document?

Thank you and best regards,
Thong Hua

**Review by test-fullautomation — APPROVED** [review]
Hi Thong,
looks good to me now.
Thank you,
 Thomas

**Pull request merged**